public/API/API_GetConsoleIDs.php

```diff
@@ -9,6 +9,20 @@
  *   object    [value]
  *    string    ID                  unique identifier of the console
  *    string    Name                name of the console
+ *    string    IconURL             site-relative path to the console icon
  */
 
-return response()->json(System::select(['ID', 'Name'])->get());
+$systems = System::select()->get();
+
+$response = [];
+
+foreach ($systems as $system) {
+    $data = [
+        'ID' => $system['ID'],
+        'Name' => $system['Name'],
+        'IconURL' => '/' . $system->getIconUrlPath(),
```

Comment:
I'd prefer this to be the absolute URL to make it clear where to load images from (media subdomain vs. main domain).

Reply:
I'm aware that games' images are referenced relative - it's a flaw I'm trying to fight for years ;)

Reply:
No problem. I'll change it.

Reply:
fwiw: this would do the trick and remove the need for the additional function in the System model:
```php
'IconURL' => $system->icon_url,
```

Reply:
Looking at the other endpoints there is no `URL` suffix.

Reply:
Changes done!

Reply:
Awesome, thanks!
`site-relative path to the console icon` could read something like `System icon URL`

Reply:
Done